src/shared/components/palette/fixed-list/fixed-list.pipe.ts

```diff
@@ -9,8 +9,11 @@ export class FixedListPipe implements PipeTransform {
   private static readonly EMPTY = '';
 
   transform(value: string, items: FixedListItem[]): any {
-    let item = items.find(i => i.code === value);
-    return item ? item.label : FixedListPipe.EMPTY;
+    if (!!items) {
+      let item = items.find(i => i.code === value);
```

Comment:
Just wanted to check that `find()` will be transpiled correctly for IE11, since `find()` isn't supported for that browser

Reply:
Need to cross check with angular material and its impact to cross browser support using find()

Reply:
This was already in place. We simply made sure that `items` wasn't null.